apps/src/lib/kits/maker/boards/microBit/ExternalLed.js

```diff
@@ -1,9 +1,9 @@
 export default class ExternalLed {
   constructor(opts) {
-    //TODO - limit to the GPIO pins on MB?
     this.board = opts.board;
     this.pin = opts.pin;
     this.isOn = false;
+    this.intervalID = null;
```

Comment:
nit: maybe `blinkIntervalID` or something so it's clear which interval this refers to?

Reply:
Will do.